src/App.js

```diff
@@ -1,23 +1,32 @@
-import React from "react";
+import React, { useState } from "react";
 import { BrowserRouter as Router, Switch, Route } from "react-router-dom";
 import { Home, Coin, Portfolio } from "pages";
 import { NavBar, NavUnder } from "components";
-import "./App.css";
+import { GlobalStyle } from "styling/GlobalStyle";
+import { darkTheme, lightTheme } from "styling/theme";
+import { ThemeProvider } from "styled-components";
+import { Container } from "App.css";
 
 function App() {
+  const [toggle, setToggle] = useState(false);
+  const handleClick = () => setToggle(toggle ? false : true);
+  const toggleTheme = () => (toggle ? lightTheme : darkTheme);
```

Comment:
do you need to make it a function?

Reply:
yeah because there would be no other way to make the event happen if I dont make a handleClick function?